lib/src/widgets/reactive_status_listenable_builder.dart

```diff
@@ -47,13 +47,11 @@ class ReactiveStatusListenableBuilder extends StatelessWidget {
   Widget build(BuildContext context) {
     var control = formControl;
     if (control == null) {
-      final form = ReactiveForm.of(context, listen: false);
+      Object? form = ReactiveForm.of(context, listen: false);
       if (form is! FormControlCollection) {
         throw FormControlParentNotFoundException(this);
       }
-
-      final collection = form as FormControlCollection;
-      control = collection.control(formControlName!);
+      control = form.control(formControlName!);
```

Comment:
Hi @kuhnroyal,

I guess here is also a matter of coding preferences, is that right or it means a real change. I see that declaring the **form** and and **Object?** let you write less code (I guess that your main goal here with this change, remove a few extra lines of code), but you're delegating on *dynamic* compilation.

Reply:
Well this puzzled me a bit as well. In theory `form as FormControlCollection` is an unnecessary cast as the compiler should be able to promote the type. But this somehow doesn't work with `final form` and I have no idea why.
And the cast is marked as unsafe.